core-rust-bridge/src/main/java/com/radixdlt/mempool/RustMempool.java

```diff
@@ -107,9 +103,7 @@ public RustMempool(Metrics metrics, StateManager stateManager) {
 
   public RawNotarizedTransaction addTransaction(RawNotarizedTransaction transaction)
       throws MempoolRejectedException {
-    final var stopwatch = Stopwatch.createStarted();
     final var result = addFunc.call(transaction);
-    metrics.mempool().addTransaction().observe(stopwatch.elapsed());
```

Comment:
_(Note from author)_

Our generic "native call" metric captures exactly the same already, for every native func, including this one.